File: Airline_Web_API/Airline_Web_API/Models/Flight.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Airline_Web_API.Models
{
    [Table("Flights")]
    public class Flight
    {
        public int Id { get; set; }
        public int DepartureAirportId { get; set; }
        public int ArrivalAirportId { get; set; }
        public int AirplaneId { get; set; }
        public DateTime DepartureDate { get; set; }
        public DateTime ArrivalDate { get; set; }
        public string Status { get; set; }

        public Airport DepartureAirport { get; set; }
        public Airport ArrivalAirport { get; set; }
        public Airplane Airplane { get; set; }

        public ICollection<Ticket> Tickets { get; set; }
    }
}

File: AdminApp/src/app/Models/Flight.ts
import { Airport } from './Airport';
import { Ticket } from './Ticket';
import { Airplane } from './Airplane';

export interface Flight {
  id: number;
  departureAirport: Airport;
  arrivalAirport: Airport;
  departureDate: Date;
  arrivalDate: Date;
  tickets?: Ticket[];
  status: string;
  airplane?: Airplane;
}
